README.md

```diff
@@ -1,2 +1,74 @@
 # Press release email generator service
-// TODO: 
+
+This microservice looks for publication tasks that have its publication-channel set to ("verzendlijsten"), are not
+started yet (adms:status) and it's publication-event has no ebucore:publicationEndDateTime yet.
+
+for every result found, it generates the email-html, updates the statuses and dates and adds an email to the outbox in
+the triplestore.
+
+## How to
+
+### Run the application in development mode
+
+For development, add a docker-compose.override.yml to your main project (app-persberichten), or add the following
+service to your existing docker-compose.override.yaml.
+(You might have to change the volume path to the root path of this application).
+
+```yaml
+services:
+  press-release-email-generator:
+    image: semtech/mu-javascript-template
+    ports:
+      - <available-port-on-device>:80
+    environment:
+      NODE_ENV: "development"
+    volumes:
+      - ../press-release-email-generator-service/:/app/
+```
+
+and add the following rules to the ``` config/delta/rules.js ```
+```json
```

Comment:
Dat is hier een `javascript` snippet (anders zouden de keys ook tussen `"` moeten staan)